### DISCUSSION TITLE
TON 1070 & 1070TI hash doesnt showin windows version

### DISCUSSION THREAD

**Discussion by aminlv** [Lolliedieb/lolMiner-releases#1133] [Q&A] [answered]
hmm not sure whats going on seems hashrate of Nvidia 1070 & 1070 ti doesnt works with dual mining ton&eth

Ethash: Average speed (15s): 29.17 Mh/s | 95.90 Mh/s | 27.68 Mh/s | 24.29 Mh/s | 24.15 Mh/s | 77.38 Mh/s Total: 278.57 Mh/s
SHA256-TON: Average speed (15s): 0.00 Mh/s | 335.50 Mh/s | 0.00 Mh/s | 0.00 Mh/s | 0.00 Mh/s | 1237.75 Mh/s Total: 1573.25 Mh/s

is there anything should i do to see the hashrate?

**Comment by themadcatdotnet** [comment]
I am seeing the same issue, but in linux.

**Reply by rodolphoAle** [reply]
![image](https://user-images.githubusercontent.com/99363815/153310686-7ad4630c-4408-4325-a4dd-eaf4f0072fe4.png)
I believe you have to leave it without PL or put it above 170 in software

**Comment by aminlv** [comment]
**update:
1.44 1070ti working with dual mining now but 1070 still not working
Ethash: Average speed (15s): 28.53 Mh/s | 95.31 Mh/s | 26.58 Mh/s | 22.69 Mh/s | 23.35 Mh/s | 83.04 Mh/s Total: 279.49 Mh/s
SHA256-TON: Average speed (15s): 0.00 Mh/s | 333.56 Mh/s | 199.19 Mh/s | 11.34 Mh/s | 0.00 Mh/s | 1785.77 Mh/s Total: 2329.87 Mh/s

**Reply by rodolphoAle** [reply]
what did you do to make this happen, why can't mine 
Ethash: Average speed (10s): 30.07 Mh/s | 30.01 Mh/s | 25.84 Mh/s | 31.02 Mh/s | 28.35 Mh/s Total: 145.30 Mh/s
SHA256-TON: Average speed (10s): 571.15 Mh/s | 1065.38 Mh/s | 155.06 Mh/s | 0.00 Mh/s | 0.00 Mh/s Total: 1791.60 Mh/s

**Reply by aminlv** [reply]
just updated to version 1.44 nothing else!
seems it was a fluke! after rebooting the rig it stop working again!
Ethash: Average speed (15s): 27.68 Mh/s | 78.44 Mh/s | 27.43 Mh/s | 22.51 Mh/s | 23.17 Mh/s | 83.15 Mh/s Total: 262.39 Mh/s
SHA256-TON: Average speed (15s): 0.00 Mh/s | 941.18 Mh/s | 0.00 Mh/s | 0.00 Mh/s | 0.00 Mh/s | 1787.32 Mh/s Total: 2728.50 Mh/s

**Reply by rodolphoAle** [reply]
![image](https://user-images.githubusercontent.com/99363815/153310770-4eaa1fff-ab88-4dc5-b850-6c118a88f921.png)
on windows you need to put at least 90% PL in software

**Reply by aminlv** [reply]
yeah son of tech is also mentioned about power Limit mine is 100% atm in windows still only my 2 3080 work properly rest randomly not working! after rig crash or reboots

**Comment by aminlv** [comment]
seriously this is weird to randomly work or not work after reboot!
on pascal gpus! i guess have to w8 for next version or something 

Ethash: Average speed (15s): 26.28 Mh/s | 72.87 Mh/s | 27.33 Mh/s | 22.90 Mh/s | 85.05 Mh/s | 113.44 Mh/s Total: 347.86 Mh/s
SHA256-TON: Average speed (15s): 0.00 Mh/s | 1093.28 Mh/s | 13.65 Mh/s | 297.71 Mh/s | 1913.37 Mh/s | 2551.13 Mh/s Total: 5869.15 Mh/s

Ethash: Average speed (15s): 26.28 Mh/s | 83.23 Mh/s | 27.11 Mh/s | 22.72 Mh/s | 86.42 Mh/s | 115.70 Mh/s Total: 361.46 Mh/s
SHA256-TON: Average speed (15s): 183.76 Mh/s | 1247.50 Mh/s | 0.00 Mh/s | 340.71 Mh/s | 1943.30 Mh/s | 2777.44 Mh/s Total: 6492.70 Mh/s

**Comment by aminlv** [comment, marked as answer]
well found why randomly in 1.44 some of pascal cards with 0 hashes on ton!
if u login or attach monitor to the rig it work just fine on dual mining 
if u dont connect randomly hash 0 from 1 or to all of them!

why is this way idk probably lol dev ppl can figure it out! on later version

**Comment by macieks28** [comment]
Hi, I have same problem but I found another temporary solution (for me better than above), that when you will start mining without PL limitation it works, and then you have to limit it again after at least 30-60 seconds since Ton starts to mine. On my rig I have three 1070, and only one is mining when starting with PL limit. With my solution it works nice. Try this
![2022-02-16_13h54_34](https://user-images.githubusercontent.com/54885090/154276492-0c851188-fc2e-48d6-a78f-2ba424431ffb.png)
